ksqldb-serde/src/main/java/io/confluent/ksql/serde/json/KsqlJsonDeserializer.java

```diff
@@ -163,6 +169,14 @@ private static Object enforceFieldType(
     }
   }
 
+  private static Object handleInt(final JsonValueContext context) {
+    if (context.schema.name() == Time.LOGICAL_NAME) {
+      return JsonSerdeUtils.toTime(context.val);
```

Comment:
I see that inside `JsonSerdeUtils.toTime` is obtained the number as long, but here we sai `handleInt`. What is the correct type we need to handle?

Reply:
see previous comment